cmd/istioctl/istioctl_test.go

```diff
@@ -155,3 +155,19 @@ func TestNewGet(t *testing.T) {
 		t.Fatalf("Could not delete world-cb: %v", err)
 	}
 }
+
+func TestInvalidRouteRule(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-route-rule.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
+		t.Fatalf("Did not fail when presented with invalid route rule")
+	}
+}
+
+func TestInvalidDestinationPolicy(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-destination-policy2.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
```

Comment:
I'd like to keep them.  Although I wrote them hoping that a call to validation from the CLI would improve my code coverage the tests validate that the CLI is, in fact, doing validation.  They have already found a bug in a test.  I request that they stay.